<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreditWindow</class>
 <widget class="QWidget" name="CreditWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-image: url(:/styles/backgroundcredit.png);
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_ac">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  background-color: transparent;
  background-image: url(:/styles/button/ac.png);
  background-repeat: no-repeat;
  background-position: center;
  width: 160px;
  height: 80px;
}

QPushButton:pressed{
  background-image: url(:/styles/button/pressed.png);
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_calc">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>390</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
  background-color: transparent;
  background-image: url(:/styles/button/frame.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 80px;
  height: 80px;
}

QPushButton:pressed{
  color: transparent;
  background-image: url(:/styles/button/pressed.png);
}</string>
   </property>
   <property name="text">
    <string>=</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>431</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
  background-color: transparent;
  background-image: url(:/styles/plot_frame.png);
  background-repeat: no-repeat;
  background-position: center;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_overpaymentOnCredit">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>160</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/frame_label.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_totalPayment">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/frame_label.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_monthlyPayment">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/frame_label.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/monthly_payment.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/overpayment_on_credit.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/total_payment.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>441</width>
     <height>441</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFrame{
  background-color: transparent;
  background-image: url(:/styles/plot_frame.png);
  background-repeat: no-repeat;
  background-position: center;
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/interest_rate.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_interestRate">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox{
  background-color: transparent;
  background-image: url(:/styles/button/frame_big.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 160px;
  height: 80px;
}

QDoubleSpinBox::up-button, QDoubleSpinBox::down-button {
  width: 0;
  height: 0;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>1.000000000000000</double>
    </property>
    <property name="maximum">
     <double>99.989999999999995</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/year.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_year">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>230</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
  background-color: transparent;
  background-image: url(:/styles/button/frame_big.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 160px;
  height: 80px;
}

QSpinBox::up-button, QSpinBox::down-button {
  width: 0;
  height: 0;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_month">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QSpinBox{
  background-color: transparent;
  background-image: url(:/styles/button/frame_big.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 160px;
  height: 80px;
}

QSpinBox::up-button, QSpinBox::down-button {
  width: 0;
  height: 0;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>1080</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/total_credit_amount.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="doubleSpinBox_totalCreditAmount">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>50</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QDoubleSpinBox{
  background-color: transparent;
  background-image: url(:/styles/button/frame_big.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 160px;
  height: 80px;
}

QDoubleSpinBox::up-button, QDoubleSpinBox::down-button {
  width: 0;
  height: 0;
}</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="minimum">
     <double>1000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>1000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>191</width>
      <height>121</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
  background-color: transparent;
  background-image: url(:/styles/label/month.png);
  background-repeat: no-repeat;
  background-position: center;
  color: black;
  width: 191px;
  height: 120px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QRadioButton" name="radioButton_diff">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>120</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
  background-color: transparent;
  background-image: url(:/styles/button/diff.png);
  background-repeat: no-repeat;
  background-position: center;
  width: 160px;
  height: 80px;
}

QRadioButton:checked{
  background-image: url(:/styles/button/pressed.png);
}

QRadioButton::indicator { width: 0px; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_ann">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>81</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton{
  background-color: transparent;
  background-image: url(:/styles/button/ann.png);
  background-repeat: no-repeat;
  background-position: center;
  width: 160px;
  height: 80px;
}

QRadioButton:checked{
  background-image: url(:/styles/button/pressed.png);
}

QRadioButton::indicator { width: 0px; }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
